src/azure-cli/azure/cli/command_modules/aro/_params.py

```diff
@@ -0,0 +1,91 @@
+from azure.cli.command_modules.aro._validators import validate_visibility
+from azure.cli.command_modules.aro._validators import validate_vnet
+from azure.cli.command_modules.aro._validators import validate_vnet_resource_group_name
+from azure.cli.command_modules.aro._validators import validate_worker_count
+from azure.cli.command_modules.aro._validators import validate_worker_vm_disk_size_gb
+from azure.cli.core.commands.parameters import name_type
+from azure.cli.core.commands.parameters import resource_group_name_type
+from azure.cli.core.commands.parameters import tags_type
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+
+
+def load_arguments(self, _):
+    with self.argument_context('aro') as c:
+        c.argument('location',
+                   validator=get_default_location_from_resource_group)
+        c.argument('resource_name',
+                   name_type,
+                   help='Name of cluster.')
+        c.argument('tags',
+                   tags_type)
+
+        c.argument('pull_secret',
+                   help='Pull secret of cluster.',
+                   validator=validate_pull_secret)
+        c.argument('domain',
+                   help='Domain of cluster.',
+                   validator=validate_domain)
+        c.argument('cluster_resource_group',
+                   help='Resource group of cluster.',
+                   validator=validate_cluster_resource_group)
+
+        c.argument('client_id',
+                   help='Client ID of cluster service principal.',
+                   validator=validate_client_id)
+        c.argument('client_secret',
+                   help='Client secret of cluster service principal.',
+                   validator=validate_client_secret)
+
+        c.argument('pod_cidr',
+                   help='CIDR of pod network.',
+                   validator=validate_cidr('pod_cidr'))
+        c.argument('service_cidr',
+                   help='CIDR of service network.',
+                   validator=validate_cidr('service_cidr'))
+
+        c.argument('master_vm_size',
+                   help='Size of master VMs.')
+
+        c.argument('worker_vm_size',
+                   help='Size of worker VMs.')
+        c.argument('worker_vm_disk_size_gb',
+                   help='Disk size in GB of worker VMs.',
+                   validator=validate_worker_vm_disk_size_gb)
+        c.argument('worker_count',
+                   help='Count of worker VMs.',
+                   validator=validate_worker_count)
+
+        c.argument('apiserver_visibility',
+                   help='API server visibility.',
+                   validator=validate_visibility('apiserver_visibility'))
+
+        c.argument('ingress_visibility',
+                   help='Ingress visibility.',
+                   validator=validate_visibility('ingress_visibility'))
+
+        c.argument('vnet_resource_group_name',
```

Comment:
Also confused like `cluster_resource_group`

Reply:
Service nature is "Bring your own VNET only". This value defines, where this VNET resides.